template/.devcontainer.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
What is this file used for?

Reply:
Moving forward, this `.devcontainer.json` file will be the new configuration file we push to student's code50 repository.